client/my-sites/upgrades/checkout/checkout.jsx

```diff
@@ -238,7 +237,8 @@ const Checkout = React.createClass( {
 				}
 			}
 		} = this.props;
-		const redirectPath = this.getCheckoutCompleteRedirectPath();
+		const redirectPath = this.getCheckoutCompleteRedirectPath(),
+			{ receipt_id: receiptId } = receipt;
```

Comment:
I misunderstood @gziolo's comment up there. He wanted me to just destructure the `receiptId` if it's there. Good catch!